isthmus/src/test/java/io/substrait/isthmus/ProtoPlanConverterTest.java

```diff
@@ -15,15 +16,46 @@ private void assertProtoRelRoundrip(String query) throws IOException, SqlParseEx
     SqlToSubstrait s = new SqlToSubstrait();
     String[] values = asString("tpch/schema.sql").split(";");
     var creates = Arrays.stream(values).filter(t -> !t.trim().isBlank()).toList();
-    Plan protoPlan1 = s.execute(query, creates);
-    io.substrait.plan.Plan plan = new ProtoPlanConverter().from(protoPlan1);
-    Plan protoPlan2 = new PlanProtoConverter().toProto(plan);
+    io.substrait.proto.Plan protoPlan1 = s.execute(query, creates);
+    String planStr1 = protoPlan1.toString();
+    Plan plan = new ProtoPlanConverter().from(protoPlan1);
+    io.substrait.proto.Plan protoPlan2 = new PlanProtoConverter().toProto(plan);
     assertEquals(protoPlan1, protoPlan2);
   }
 
+  private io.substrait.proto.Plan getProtoPlan(String query1)
+      throws IOException, SqlParseException {
+    SqlToSubstrait s = new SqlToSubstrait();
+    String[] values = asString("tpch/schema.sql").split(";");
+    var creates = Arrays.stream(values).filter(t -> !t.trim().isBlank()).toList();
+    return s.execute(query1, creates);
+  }
+
   @Test
   public void aggregate() throws IOException, SqlParseException {
-    assertProtoRelRoundrip("select count(DISTINCT L_ORDERKEY),sum(L_ORDERKEY) from lineitem");
+    assertProtoRelRoundrip("select count(L_ORDERKEY),sum(L_ORDERKEY) from lineitem");
```

Comment:
We should add proto roundtrip tests to the core module. Relying on the SQL to Substrait code substantially increases this kind of bug.

Reply:
Added core test for aggregate function invocation roundtrip.